### PR TITLE
Ran pre-commit, added licenses

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Review by FoamyGuy — COMMENTED** [review]
some of these have different copyrights listed in the code file than in then license file. three.js, jquery, and bootstrap were the ones that caught my eye

**Comment by dherrada** [comment]
@FoamyGuy thanks for noticing that! I'll get that fixed sometime today hopefully

File: examples/bno055_webgl_demo/static/css/bootstrap.css.license [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```
Comment (FoamyGuy) [inline examples/bno055_webgl_demo/static/css/bootstrap.css.license:1]: bootstrap.css is still listed as copyright 2021 ladyada here.

The bootstrap license page shows copyright twitter 2018 https://getbootstrap.com/docs/4.0/about/license/

File: examples/bno055_webgl_demo/static/css/bootstrap.css.map.license [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```
Comment (FoamyGuy) [inline examples/bno055_webgl_demo/static/css/bootstrap.css.map.license:1]: I am less sure about .map files I don't know if those are part of the core bootstrap project or if they were something added to this project. if they are core bootstrap I would think they should be copyright 2018 twitter also.

File: examples/bno055_webgl_demo/static/css/bootstrap.min.css.license [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```
Comment (FoamyGuy) [inline examples/bno055_webgl_demo/static/css/bootstrap.min.css.license:1]: bootstrap shows 2021 ladyada here, should be 2018 twitter I think

**Review by FoamyGuy — COMMENTED** [review]
I found a few more bootstrap copyrights that need changed I think.

jquery, three.js and everything else is looking good to me now. Ready to merge after the last few bootstrap ones I think.

File: examples/bno055_webgl_demo/static/css/bootstrap.css.license (hunk 1 above)
Comment (FoamyGuy) [reply examples/bno055_webgl_demo/static/css/bootstrap.css.license]: latest commit fixes all of the remaining bootstrap licenses that I found.

**Review by FoamyGuy — APPROVED** [review]

**Pull request merged**